pagopa/pn-frontend-e2e
# Pn 9249 9250 9251 9252 9253

## Short description

Common PR for tests from 9249 to 9253

## List of changes proposed in this pull request

- Implemented test from 9249 to 9253, test 9252 refused, cannot be implemented as TA
- Added some steps and improved some existing steps to make them general
- Tests moved in a single file

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@invioNotificaDigitaleADomicilioImpostato"` and verify if the tests work fine.

Run test on AWS: https://eu-south-1.console.aws.amazon.com/codesuite/codebuild/151559006927/projects/pn-RunFeE2eTestsCodeBuildProject/build/pn-RunFeE2eTestsCodeBuildProject%3A07a5b702-f577-4b33-a289-2cbeec1a4c7d?region=eu-south-1
See the report, all tests have passed.
